lib/assets/javascripts/cable.coffee

```diff
@@ -0,0 +1,19 @@
+#= require_self
+#= require cable/consumer
+
+@Cable =
+  PING_IDENTIFIER: "_ping"
+
+  createConsumer: (url) ->
+    new Cable.Consumer websocketUrl(url)
+
+  websocketUrl = (url) ->
+    if url and not /^wss?:/i.test(url)
+      a =  document.createElement('a')
+      a.href = url
+      # ie fix for relative urls
```

Comment:
Fixes them how? What happens without `a.href = a.href`? That's what I'd like to document.